addons/io/org.openhab.io.neeo/src/main/java/org/openhab/io/neeo/NeeoConstants.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.io.neeo;
+
+import java.io.File;
+
+import org.eclipse.smarthome.config.core.ConfigConstants;
+
+/**
+ * The constants class for the NEEO Connector
+ *
+ * @author Tim Roberts - Initial contribution
+ */
+public class NeeoConstants {
+
+    /** Constants used for the Web APP */
+    public static final String WEBAPP_PREFIX = "/neeo";
+    public static final String WEBAPP_STATUS_PREFIX = "/neeostatus";
+
+    /** The MDNS type for neeo */
+    public static final String NEEO_MDNS_TYPE = "_neeo._tcp.local.";
+
+    /** The constants used for configuration */
+    public static final String CFG_EXPOSE_ALL = "exposeAll";
+    public static final String CFG_IPADDRESSES = "ipAddresses";
+
+    /** The name of the adapter */
+    public static final String ADAPTER_NAME = "openHAB";
+
+    /** The default port the brain listens on. */
+    public static final int DEFAULT_BRAIN_PORT = 3000;
+
+    /** The default protocol for the brain. */
+    public static final String PROTOCOL = "http://";
+
+    /** The device definitions file name */
+    public static final String FILENAME_DEVICEDEFINITIONS = ConfigConstants.getUserDataFolder() + File.separator
+            + "neeo" + File.separator + "neeodefinitions.json";
+
+    /** The search threshold value */
+    public static final double SEARCH_MATCHFACTOR = 0.5;
+
+    /** Various brain URLs */
+    public static final String REGISTER_SDK_ADAPTER = "/v1/api/registerSdkDeviceAdapter";
```

Comment:
I'm going to do this but like the prior comment - may be a waste of time ("v1" was introduced in the latest firmware as well - not sure if it will stay, go to "v2" or whatever in the future).